docs/faq.md

```diff
@@ -21,35 +21,35 @@ sudo pip install cloud4rpi
 
 ## Raspberry Pi does not recognize my 1-Wire device
 
-If you experience issues with the 1-Wire interface on Raspberry Pi, follow the instructions below.
+Follow the instructions below if you experience issues with the 1-Wire interface on Raspberry Pi.
 
 1. Run `sudo raspi-config` and make sure that the 1-Wire interface is enabled (**Interfacing Options** | **1-Wire** menu).
 2. Add the `dtoverlay=w1-gpio` string to your `/boot/config.txt` file (`echo dtoverlay=w1-gpio | sudo tee -a /boot/config.txt`).
 3. Reboot (`sudo reboot` command).
-4. Double-check the wiring and your device's operational voltage. The 1-Wire **DATA** bus should be connected to **GPIO4** (Pin 7) and pulled up to **VCC**.
```

Comment:
That's incorrect. "Pulling up" means "connecting a pin to a positive voltage through a resistor" https://en.wikipedia.org/wiki/Pull-up_resistor